includes/class-sensei-content-drip.php

```diff
@@ -14,6 +14,15 @@
  * @author WooThemes
  * @since 1.0.0
  *
+ * @property Scd_Ext_Settings $settings
```

Comment:
For these we should probably add explicit public properties onto the class and then put the docblock notation above the properties, such as:

```php
	/**
	 * Stores the singleton settings object.
	 * 
	 * @var Scd_Ext_Settings
	 */
	public $settings;
```

This is probably outside of the scope of this PR, though.